mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/DatasourceSimulator.java

```diff
@@ -43,42 +37,74 @@ public boolean connect() {
         return true;
     }
 
-    public boolean isConnected() {
-        return this.connected;
-    }
-
     public void disconnect() {}
 
-    public int getTagCount() {
-        return this.tagCount;
+    public Iterable<Element> getDatabases() {
+        int databaseCount = RNG.nextInt(10);
+        return IntStream.range(1, databaseCount + 1)
```

Comment:
To be more similar with csharp we should use IntStream.rangeClosed (to avoid +1 at the end interval)